doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -255,6 +255,7 @@ Serial LTE modem
 nRF5340 Audio
 -------------
 
+* The SoftDevice Controller is the default controller for the build script.
```

Comment:
```suggestion
* The Nordic Semiconductor SoftDevice Controller (:ref:`ug_ble_controller_softdevice`) has now replaced the LE Audio Controller Subsystem for nRF5340 (The :ref:`_lib_bt_ll_acs_nrf53_readme`) as the default controller. For NCS 2.6.0 and future NCS releases, users must transition to and use the SoftDevice Controller. There should be no negative impact on performance for users of the nRF5340 Audio application. This change will enable use of standard tools for building and configuring the controller. Note that the SoftDevice controller will for some time not have a QDID for ISO operation.
```

Reply:
Most of the information from @koffes edit should go to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/migration/migration_guide_2.6.html and be added there in an entry about the Audio applications.

I suggest keeping it short and simple here in the changelog:
```suggestion
* Changed the default controller from the :ref:`_lib_bt_ll_acs_nrf53_readme` to the :ref:`ug_ble_controller_softdevice`. Read :ref:`migration_2.6` for information about how this affects your application.
```

And then in the migration guide doc:
```
* The :ref:`nrf53_audio_app` have changed the default controller from the :ref:`_lib_bt_ll_acs_nrf53_readme` to Nordic Semiconductor's :ref:`ug_ble_controller_softdevice`. Make sure to transition to using the SoftDevice Controller. There should be no negative impact on performance for the nRF5340 Audio applications. This change will enable use of standard tools for building and configuring the controller. The SoftDevice Controller will for some time not have a QDID for the ISO operation. 
```